app/src/main/res/layout/activity_forgot_password_page.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Make forgot password page look more like this
Where the input is below the title Calvin Email:
Then you can make the hint for the input be Enter Calvin Email 
Make the text that is typed in be white instead of black
Make the button be red instead of grey
![image](https://user-images.githubusercontent.com/38587739/67540343-04b41380-f6b3-11e9-9201-71ef964f5449.png)